docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
+
+If the Domain Mapping feature is installed, you can serve
+custom domains backed by Knative Services. For example, if you
+own the "MyDomain.com" domain name, you can point DNS at your
+Knative cluster, and have this domain name be served by a
+Knative Service.
+
+## Before you begin
+
+To map a custom domain to a Knative Service, you should first [create a Knative
+Service](https://knative.dev/docs/serving/creating-services/).
+
+## Creating a Domain Mapping
+
+To create a mapping from a custom domain to a Knative Service, you need to
+create a YAML file that defines a Domain Mapping.
+This YAML file specifies the domain name to map and the Knative Service to use
+to service requests.
+
+You will also need to point the domain name at your Knative cluster using the
+tools provided by your domain registrar.
+
+### Procedure
+
+1. Create a new file named `domainmapping.yaml` containing the following information.
+  ```yaml
+  apiVersion: serving.knative.dev/v1alpha1
+  kind: DomainMapping
+  metadata:
+   name: mydomain.com
```

Comment:
Is there a spec.domain field? Because I would be very disappointed if the actual domain value is the name of the CR

Reply:
It's intentionally the name of the DM to prevent collisions. Here's one of the conversations in the design doc where we decided this: https://docs.google.com/document/d/1yXSuKpQqZM5tczBFpPG8TGQoWSkvmoA17nPwDd1UxEI/edit?disco=AAAAG2yt488.